### PR TITLE
[WIP] Use UNLOGGED TABLES on PostgreSQL

### LINKED ISSUE #13: Use Unlogged Tables in PostgreSQL
Unlogged Tables was first introduced at PostgreSQL 9.1. From [release notes](https://www.postgresql.org/docs/9.1/release-9-1.html#AEN110548):

> * Support unlogged tables using the UNLOGGED option in CREATE TABLE (Robert Haas)
> 
> Such tables provide better update performance than regular tables, but are not crash-safe: their contents are automatically cleared in case of a server crash. Their contents do not propagate to replication slaves, either.

Wonder if can can add this statement here https://github.com/knadh/sql-jobber/blob/master/backends/sqldb.go#L282 ? And perhaps add a new config option  `unlogged = true` that works just for `dbType=postgres`

### PR DESCRIPTION
**Pull request by fabriziomello**
WIP example of UNLOGGED TABLES in PostgreSQL (resolve #13)

### CODE REVIEW

File: backends/sqldb.go [hunk 1]
```diff
@@ -277,9 +278,12 @@ func (s *sqlDB) createTableSchema(cols []string, colTypes []*sql.ColumnType) ins
 		fields[i] = fmt.Sprintf(`"%s" %s`, cols[i], typ)
 	}
 
+	if s.dbType == dbTypePostgres {
+		unlogged = "UNLOGGED "
+	}
 	return insertSchema{
 		dropTable:   `DROP TABLE IF EXISTS "%s";`,
-		createTable: fmt.Sprintf(`CREATE TABLE IF NOT EXISTS "%%s" (%s);`, strings.Join(fields, ",")),
+		createTable: fmt.Sprintf(`CREATE %sTABLE IF NOT EXISTS "%%s" (%s);`, unlogged, strings.Join(fields, ",")),
```
Comment (knadh) [inline backends/sqldb.go:286]: It's fine to leave a space after `%s` for readability.

**Comment by fabriziomello** [comment]
@knadh I'm not a golang developer but I did a refactor as requested

**Comment by knadh** [comment]
> I'm not a golang developer but I did a refactor as requested

Nonetheless, the PR is good! Made a few minor cosmetic changes

File: backends/sqldb.go [hunk 2]
```diff
@@ -285,7 +286,10 @@ func (s *sqlDB) createTableSchema(cols []string, colTypes []*sql.ColumnType) ins
 		fields[i] = fmt.Sprintf(`"%s" %s`, cols[i], typ)
 	}
 
-	if s.opt.DbType == dbTypePostgres && s.opt.UnloggedTable {
+	// If the DB is Postgres, optionally create an "unlogged" table that disables
+	// WAL, improving performance of throw-away cache tables.
+	// https://www.postgresql.org/docs/9.1/sql-createtable.html
```
Comment (fabriziomello) [inline backends/sqldb.go:291]: Here the best is use the `current` link for docs. I mean:

https://www.postgresql.org/docs/current/sql-createtable.html

PS: 9.1 is old and deprecated

Comment (knadh) [reply backends/sqldb.go:291]: Woops, fixed: https://github.com/knadh/sql-jobber/commit/b2536b9a614dd35292c81325f7fb02b8c047778d

**Pull request merged**